pretalx_broadcast_tools/templates/pretalx_broadcast_tools/room_info.html

```diff
@@ -7,7 +7,7 @@
         <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
         <title>{{ request.event.name }} room info</title>
         {% compress js %}
-            <script src="{% static "vendored/jquery-3.1.1.js" %}"></script>
+            <script src="{% static "js/jquery.js" %}"></script>
```

Comment:
I have pushed some commits that remove the need for jQuery altogether. Will merge this after the pretalx commit is merged, then remove those lines.